<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PhotonFinder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New_Tab"/>
    <addaction name="actionDuplicate_Tab"/>
    <addaction name="action_Close_Tab"/>
    <addaction name="separator"/>
    <addaction name="action_Create_Database"/>
    <addaction name="action_Open_Database"/>
    <addaction name="action_Create_Backup"/>
    <addaction name="separator"/>
    <addaction name="action_Scan_Libraries"/>
    <addaction name="action_Manage_Libraries"/>
    <addaction name="separator"/>
    <addaction name="action_Settings"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_View_Log"/>
    <addaction name="separator"/>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>F&amp;ilter</string>
    </property>
    <addaction name="actionExposure"/>
    <addaction name="actionCoordinates"/>
    <addaction name="actionDate"/>
    <addaction name="actionTelescope"/>
    <addaction name="actionBinning"/>
    <addaction name="actionGain"/>
    <addaction name="actionTemperature"/>
    <addaction name="separator"/>
    <addaction name="actionHeader_Text"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionShow_location"/>
    <addaction name="actionSelect_path"/>
    <addaction name="separator"/>
    <addaction name="action_Export_Data"/>
    <addaction name="actionPlate_solve_files"/>
    <addaction name="actionPlate_Solve_Astrometry_net"/>
    <addaction name="separator"/>
    <addaction name="actionFind_matching_darks"/>
    <addaction name="actionFind_matching_flats"/>
   </widget>
   <widget class="QMenu" name="menuReport">
    <property name="title">
     <string>Report</string>
    </property>
    <addaction name="actionList_Files"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuFilter"/>
   <addaction name="menu_Tools"/>
   <addaction name="menuReport"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_View_Log">
   <property name="text">
    <string>View &amp;Log</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_New_Tab">
   <property name="text">
    <string>&amp;New Tab</string>
   </property>
  </action>
  <action name="action_Close_Tab">
   <property name="text">
    <string>&amp;Close Tab</string>
   </property>
  </action>
  <action name="action_Manage_Libraries">
   <property name="text">
    <string>Manage &amp;Libraries</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="action_Scan_Libraries">
   <property name="text">
    <string>&amp;Scan Libraries</string>
   </property>
  </action>
  <action name="actionExposure">
   <property name="text">
    <string>Exposure</string>
   </property>
  </action>
  <action name="actionCoordinates">
   <property name="text">
    <string>Coordinates</string>
   </property>
  </action>
  <action name="actionDate">
   <property name="text">
    <string>Date</string>
   </property>
  </action>
  <action name="actionTelescope">
   <property name="text">
    <string>Telescope</string>
   </property>
  </action>
  <action name="actionBinning">
   <property name="text">
    <string>Binning</string>
   </property>
  </action>
  <action name="actionGain">
   <property name="text">
    <string>Gain</string>
   </property>
  </action>
  <action name="actionTemperature">
   <property name="text">
    <string>Temperature</string>
   </property>
  </action>
  <action name="action_Export_Data">
   <property name="text">
    <string>Export file copies</string>
   </property>
  </action>
  <action name="action_Create_Backup">
   <property name="text">
    <string>&amp;Backup Database</string>
   </property>
  </action>
  <action name="action_Create_Database">
   <property name="text">
    <string>Create &amp;Database</string>
   </property>
  </action>
  <action name="action_Open_Database">
   <property name="text">
    <string>&amp;Open Database</string>
   </property>
  </action>
  <action name="actionDuplicate_Tab">
   <property name="text">
    <string>Duplicate Tab</string>
   </property>
  </action>
  <action name="actionFind_matching_darks">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Find matching darks</string>
   </property>
  </action>
  <action name="actionFind_matching_flats">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Find matching flats</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionShow_location">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show location</string>
   </property>
  </action>
  <action name="actionSelect_path">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select path</string>
   </property>
  </action>
  <action name="actionPlate_solve_files">
   <property name="text">
    <string>Plate solve (ASTAP)</string>
   </property>
  </action>
  <action name="actionPlate_Solve_Astrometry_net">
   <property name="text">
    <string>Plate Solve (Astrometry.net)</string>
   </property>
  </action>
  <action name="actionList_Files">
   <property name="text">
    <string>List Files</string>
   </property>
  </action>
  <action name="actionHeader_Text">
   <property name="text">
    <string>Header Text</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tabWidget</sender>
   <signal>tabCloseRequested(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>close_search_tab(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Export_Data</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>export_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_New_Tab</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>new_search_tab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Close_Tab</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close_current_search_tab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Manage_Libraries</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>manage_library_roots()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Settings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_settings_dialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Scan_Libraries</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>scan_libraries()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExposure</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>add_exposure_filter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDate</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>add_datetime_filter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_View_Log</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>view_log()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCoordinates</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>add_coordinates_filter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTelescope</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>add_telescope_filter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBinning</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>add_binning_filter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGain</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>add_gain_filter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTemperature</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>add_temperature_filter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Create_Backup</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>create_backup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Create_Database</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>create_database()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Open_Database</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_database()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDuplicate_Tab</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>dup_search_tab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFind_matching_darks</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>find_matching_darks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFind_matching_flats</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>find_matching_flats()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_About</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_about_dialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_tab_switch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_File</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_selected_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow_location</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_file_location()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelect_path</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>select_path_in_tree()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPlate_solve_files</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>plate_solve_files()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPlate_Solve_Astrometry_net</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>plate_solve_files_astrometry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionList_Files</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>report_list_files()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHeader_Text</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>add_header_text_filter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>close_search_tab(int)</slot>
  <slot>new_search_tab()</slot>
  <slot>close_current_search_tab()</slot>
  <slot>manage_library_roots()</slot>
  <slot>open_settings_dialog()</slot>
  <slot>scan_libraries()</slot>
  <slot>add_datetime_filter()</slot>
  <slot>add_exposure_filter()</slot>
  <slot>add_coordinates_filter()</slot>
  <slot>add_telescope_filter()</slot>
  <slot>add_binning_filter()</slot>
  <slot>add_gain_filter()</slot>
  <slot>add_temperature_filter()</slot>
  <slot>add_header_text_filter()</slot>
  <slot>view_log()</slot>
  <slot>export_data()</slot>
  <slot>create_backup()</slot>
  <slot>create_database()</slot>
  <slot>open_database()</slot>
  <slot>dup_search_tab()</slot>
  <slot>find_matching_darks()</slot>
  <slot>find_matching_flats()</slot>
  <slot>on_tab_switch()</slot>
  <slot>open_selected_file()</slot>
  <slot>show_file_location()</slot>
  <slot>select_path_in_tree()</slot>
  <slot>plate_solve_files()</slot>
  <slot>plate_solve_files_astrometry()</slot>
  <slot>report_list_files()</slot>
 </slots>
</ui>
